client/src/components/Add.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { connect } from 'react-redux';
+import * as actions from "../actions";
+import { ADD_FORM } from "../util/forms";
+import FORM_FIELDS from './addFormFields';
+import TextInput from "./forms/TextInput";
+import _ from 'lodash';
+import { Link } from "react-router-dom";
+
+class Add extends Component {
```

Comment:
Lets rename the file, `Add` doesn't really say what this component does.

```suggestion
class AddProductForm extends Component {
```

Reply:
Good idea. I'll update it.